src/components/common/Header.vue

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
Didn't we think we should use the header from sbc-common-components?

Reply:
You are correct sir, I implemented the common components but forgot to remove this placeholder. Will remove!